src/vm/codeman.cpp

```diff
@@ -3020,7 +3020,7 @@ void * EEJitManager::allocCodeFragmentBlock(size_t blockSize, unsigned alignment
     {
         CrstHolder ch(&m_CodeHeapCritSec);
 
-        mem = (TADDR) allocCodeRaw(&requestInfo, sizeof(TADDR), blockSize, alignment, &pCodeHeap);
+        mem = (TADDR) allocCodeRaw(&requestInfo, sizeof(CodeHeader), blockSize, alignment, &pCodeHeap);
```

Comment:
@jkotas When I enable WIN64EXCEPTIONS, I encountered assert failure during method table generation. While digging that issue, I found that the changes in the last field of CodeHeader overwrites the previous method table.

I'm still wondering why that issue doesn't happen unless WIN64EXCEPTIONS is defined.